### PR TITLE
core: frontend: add proxy for APIs in dev mode

### PR DESCRIPTION
**Pull request by Williangalvani**
This allows running the frontend in the local machine while still talking to the actual API running in the companion.

### CODE REVIEW

File: core/frontend/package.json [hunk 1]
```diff
@@ -3,6 +3,46 @@
...
+        "^/wifi-manager": {
+          "target": "http://companion.local/"
+        },
+        "^/cable-guy": {
+          "target": "http://companion.local/"
+        },
+        "^/helper": {
+          "target": "http://companion.local/"
+        },
+        "^/logviewer": {
```
Comment (patrickelectric) [inline core/frontend/package.json:41]: Can we make this list alphabetic ?

Comment (Williangalvani) [reply core/frontend/package.json:41]: This is following the list in nginx.conf. we should probably make both

Comment (rafaellehmkuhl) [reply core/frontend/package.json:41]: Agree.

File: core/frontend/package.json [hunk 2]
```diff
@@ -3,6 +3,46 @@
   "version": "0.1.0",
   "private": true,
   "vue": {
+    "devServer": {
```
Comment (rafaellehmkuhl) [inline core/frontend/package.json:6]: How should I use it?

Comment (Williangalvani) [reply core/frontend/package.json:6]: yarn run serve will use this

Comment (Williangalvani) [reply core/frontend/package.json:6]: then you should be able to open localhost:8080 and everything should work automagically.
you can delete some of this lines or point it to your machine if you want to run some apis locally.

Comment (rafaellehmkuhl) [reply core/frontend/package.json:6]: Cool!

**Review by rafaellehmkuhl — APPROVED** [review]

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**